tests/main/try/task.yaml

```diff
@@ -66,24 +63,27 @@ execute: |
 
     echo "Given a buildable snap which access confinement-enabled network resources in a known directory"
     echo "When try is executed on that directory"
-    snap try $TESTSLIB/snaps/network-consumer
+    snap try "$TESTSLIB"/snaps/network-consumer
 
     echo "Then the snap is able to access the network resource"
-    network-consumer http://127.0.0.1:$PORT | MATCH "ok"
+    network-consumer http://127.0.0.1:"$PORT" | MATCH "ok"
 
     echo "====================================="
     n=test-snapd-classic-confinement
     s=$TESTSLIB/snaps/$n
     echo "Given a buildable snap with classic confinement:"
     echo " - you can't try it without --classic:"
-    snap try $s && exit 1 || true
-    snap try --jailmode $s && exit 1 || true
-    snap try --devmode $s && exit 1 || true
+    #shellcheck disable=SC2015
```

Comment:
instead of the disable, we could instead rewrite `foo && bar || baz` as `( foo && bar ) || baz`, to make shellcheck happy but also make it more obvious to the reader?

Reply:
Oh well, or just `! snap try ..`